src/Pos/Ssc/Extra/Logic.hs

```diff
@@ -73,18 +76,14 @@ sscRunLocalQuery action = do
 sscRunLocalSTM
     :: forall ssc m a.
        (MonadSscMem ssc m, MonadIO m, WithLogger m)
-    => (LoggerNameBox (PureLogger (StateT (SscLocalData ssc) STM)) a) -> m a
+    => NamedPureLogger (StateT (SscLocalData ssc) STM) a -> m a
 sscRunLocalSTM action = do
-    loggerName <- getLoggerName
     localVar <- sscLocal <$> askSscMem
```

Comment:
(I mean why we know for sure that it won't be touched)
(I understand it, but it took some time)